### DISCUSSION TITLE
Can't flash printer

### DISCUSSION THREAD

**Discussion by PapaPlintus** [dz0ny/klipper-prusa-mk3s#51] [General]
I followed the steps to get klipper running on my mk3s but I keep getting 'mcu': Unable to connect error. The screen on my mk3s says it's downloading firmware but it just sits there forever. Is it just slow or am I doing something wrong?

**Comment by nakwada** [comment]
You need to edit the mcu section in printer.cfg file from Mainsail as follow:

```
[mcu]
# serial: /dev/serial0 # If you are using internal RPI serial port
serial: /dev/ttyACM0 # If you are using RPI via USB connection to printer
```
